scripts/HOWTO-distribution.md

```diff
@@ -17,8 +17,44 @@ People building releases for distribution need permission to directly operate wi
 
 See the image at the bottom for how Git is used during this process. You should have your Git remotes set up in the [recommended fashion](https://www.jmri.org/help/en/html/doc/Technical/gitdeveloper.shtml#repo).
 
+To check, run this command:
+```
+        git remote -v
+```
+If everything is correct, it will show:
+```
+        github	https://github.com/JMRI/JMRI.git (fetch)
+        github	https://github.com/JMRI/JMRI.git (push)
+        origin	https://github.com/JMRI/JMRI.git (fetch)
+        origin	https://github.com/yourname/JMRI.git (push)
+        upstream	https://github.com/JMRI/JMRI.git (fetch)
```

Comment:
I don't think you actually need the `upstream` lines for the command line.  That might be a GitHub Desktop thing.

Reply:
Thanks. Fixed.